### ISSUE TITLE
Readme Typo Fix

### ISSUE THREAD

**Issue by arshadkazmi42** [bibekluitel/JMON#39]
Change this 
`jmon.get('JMON.name').isMOdified();`
to
`jmon.get('JMON.name').isModified();`

In modified capital `O` is added. It should be small

**Comment by sinharaksh1t** [comment]
I'm helping my friend start with git and open source contributions. If no one has taken it before he comments, I'd like to save this one for him. He will comment within the next 3-5 hours when he can start with this.

**Comment by arshadkazmi42** [comment]
@sinharaksh1t It's first come first serve basis. If some else claims this, then we might have to give it to them.
If you friend is interested in this. You can follow these twitter accounts
https://twitter.com/first_issues // tweets good first issues
https://twitter.com/first_tmrs_only //tweets first-timers-issue

**Comment by sinharaksh1t** [comment]
Yup, I understand. And yes, I already started following those accounts and also asked my friend to do so. Thanks though!
I'll also follow your project closely and pick on issues which could be of interest to me or him. Thanks for helping out new coders cope up with the fear of OSS contributions so easily. Cheers 🥂

**Comment by arshadkazmi42** [comment]
We are also planning on starting a group or channel, for new contributors to help them get started.
If you are interested in joining it, you can DM me your email ID on twitter. I will send you a invite.
https://twitter.com/arshadkazmi42

**Comment by sinharaksh1t** [comment]
Sure. DM'd you @arshadkazmi42

**Comment by krishnashum** [comment]
Can I work on this ?

**Comment by arshadkazmi42** [comment]
@krishnashum Yes you can.
